doc/src/user_guide.rst

```diff
@@ -515,25 +514,24 @@ Parameterises the time-integration scheme used by the solver with
 
                *float*
 
-            - ``min-fact`` --- minimum factor that the local
+            - ``min-fact`` --- minimum factor by which the local
               pseudo time-step can change between iterations
               (suitable range 0.98-0.998)
 
                *float*
 
-            - ``max-fact`` --- maximum factor that the local
+            - ``max-fact`` --- maximum factor by which the local
               pseudo time-step can change between iterations
               (suitable range 1.001-1.01)
 
                *float*
 
-            - ``pseudo-dt-max-fact`` --- maximum permissible
```

Comment:
Should this still be suffixed?

Reply:
Removal is intentional, and thus we rely on the explanation to tell user that this is a multiplier as opposed to an absolute value (as is the case for the physical analogue). Reason for removal was due to associations of max-fact above with incremental time-step changes. But debatable - there is not a completely elegant solution ...

Reply:
Any reason we do not just make it an absolute value?

Reply:
I defer to @nloppi

Reply:
So my view here is that we either come up with a sensible name for this constant or move to absolute values with the current name (which is consistent with the standard time stepper).

Reply:
I'm personally not averse to moving to an absolute value, and leaving the user to do the maths. Also has advantage of being consistent with the current stepper.

Reply:
Sorry, I prefer a relative value. I think having an absolute value with a relative default value is more confusing than a slightly akward name.

Reply:
What do you mean by a relative default value?  If we went with an absolute value I would see the key becoming mandatory with no default.

Reply:
self.dtaumax = self.cfg.getfloat(sect, 'pseudo-dt-max', 3.0*self._dtau)

A factor of 3 works great in most cases, and I think it should be the default. We can't really specify an absolute default as in dtmax

self.dtmax = self.cfg.getfloat(sect, 'dt-max', 1e2)

Reply:
So due to the way default values are handled in PyFR this may be somewhat confusing especially if one tries to restart or reuse the resulting config file later on (since an absolute value will now be hardcoded).  If the value is absolute then we can not reasonably have a variable default.

To this end we should look at getting a better name for this factor that is consistent with the rest of PyFR.

Reply:
Maybe pseudo-dt-limit (or rlimit) and explanation in the user guide?

Reply:
What about -mult?

Reply:
I've updated the PR with -mult.